### ISSUE TITLE
Reconfigure Model and Datamodule

### ISSUE THREAD

**Issue by ruhyadi** [ruhyadi/yolo3d-lightning#8]
Reconfiguring model and datamodule to be able get better results

**Comment by ruhyadi** [comment]
Success reconfigure and training with new model, but results more worse
[commit](https://github.com/ruhyadi/yolo3d-lightning/commit/a6a8eb0753e4a3a026e342f0a6b14dcfb47c2c8e)

Car AP(Average Precision)@0.70, 0.50, 0.50:
bbox AP:76.38, 62.40, 55.23
bev  AP:3.86, 3.20, 2.91
3d   AP:1.90, 1.52, 1.52
aos  AP:75.33, 61.36, 54.01

**Comment by ruhyadi** [comment]
Success train on new model and datamodule, including new loss function. Look promising.
- [commit](https://github.com/ruhyadi/yolo3d-lightning/commit/0c7c3183f004f3429f0425f1b0741ce390e40554) 

wandb: Run summary:
wandb:               epoch 9
wandb:     train/conf_loss 0.35102
wandb:      train/dim_loss 0.15514
wandb:          train/loss 0.5902
wandb:   train/orient_loss 0.36526
wandb:    train/theta_loss 0.49712
wandb: trainer/global_step 609
wandb:        val/avg_loss 0.58785
wandb:       val/conf_loss 0.34682
wandb:        val/dim_loss 0.14274
wandb:            val/loss 0.58468
wandb:     val/orient_loss 0.38055
wandb:      val/theta_loss 0.49904

Car AP(Average Precision)@0.70, 0.50, 0.50:
bbox AP:76.38, 62.40, 55.23
bev  AP:2.01, 1.90, 1.90
3d   AP:1.69, 1.69, 1.69
aos  AP:70.35, 55.98, 48.91

**Comment by ruhyadi** [comment]
Get error on orientation inference:

![image](https://user-images.githubusercontent.com/39303602/188378049-d6022a0c-007e-4c12-8f4f-8e27c8d62661.png)

**Comment by ruhyadi** [comment]
Found the problems:
https://github.com/ruhyadi/yolo3d-lightning/blob/0c7c3183f004f3429f0425f1b0741ce390e40554/src/datamodules/components/kitti_dataset.py#L514-L516

**Comment by ruhyadi** [comment]
Fixing the problems by disabeling orientation augmentation, but the result not good enough.

![image](https://user-images.githubusercontent.com/39303602/188385621-8f6f1484-2435-4f76-9c05-7352e4f2128a.png)
